### DISCUSSION TITLE
Mentor for the DEV_G project

### DISCUSSION THREAD

**Discussion by maha-sachin** [ashutoshgithubs/iemhacks_DEV-G#39] [General]
Hi, I am Maha. 
I'm thrilled to announce that I'm available to serve as a mentor for the DEV_G project! If you have any questions or need guidance, feel free to tag me, and I'll be more than happy to assist. Let's collaborate and make this project a huge success together! 💪

Looking forward to working with all of you!

**Comment by ashutoshgithubs** [comment]
Hi @maha-sachin, That's wonderful news! I'm excited to have you as a mentor for the Dev-G project. Your support will be invaluable. Can you please tag me on discord as I'm unable to find you there.
Looking forward to collaborating with you!

**Reply by maha-sachin** [reply]
hi @ashutoshgithubs ,
Thank you for your warm welcome! I'll be working from the Canada EST time zone, so sometimes my responses may be delayed. Please don't take it the wrong way. Sure I will tag you in Discord can you share your discord name?

**Comment by ashutoshgithubs** [comment]
@maha-sachin My discord id :  _ashubhai
